ProblemSolvingMentoringSession/Part 1/AbsoluteDifferenceChallenge/Solvings/AbsoluteDifferenceChallenge_Yurchenko.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  AbsoluteDifferenceChallenge_Yurchenko.swift
+//  ProblemSolvingMentoringSession
+//
+//  Created by Владислав Юрченко on 17.05.2023.
+//  Copyright © 2023 Almost Engineer. All rights reserved.
+//
+
+import Foundation
+final class AbsoluteDifferenceChallenge_Yurchenko: AbsoluteDifferenceChallengeProtocol {
```

Comment:
` func solve(_ n: Int) -> Int {
var result = Int()

    if n == 0 {
        return 51
    }
    
    result = abs(51 - n)
    
    if n > 51 {
        result = result * 2
    }
    return result
}`


You don't need if statement about which of number are bigger, if you need just absolute difference. Than you can find difference once